### DISCUSSION TITLE
Welcome to gui-python-gtk Discussions!

### DISCUSSION THREAD

**Discussion by natorsc** [natorsc/py-gtk#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peterjosvai** [comment]
hi, 

Your tutorial-kind-of example collection came with perfect timing! :) 

Not only for me, studying GTK from scratch.. having started with GTK3... and now moving over to 4... (on a very basic level)
But for everyone...

Cause since version 4.1 lot of things have been dropped... like FileChooserNative... 
which means most tutorials / collections being outdated...
and GTK4 hard to access

Your repo here is a treasure :)  

Thanks very much!!

Peter
